jujugui/static/gui/src/app/templates/storefront-search.handlebars

```diff
@@ -0,0 +1,96 @@
+        </div>
+      {{/if}}
+      <div class="list-block__filters">
+        <nav class="six-col list-block__type">
+          <ul>
+            <li class="{{#compare currentType 'all'}}selected{{/compare}}" data-type="all" tabindex="0" role="button">All</li>
+            <li class="{{#compare currentType 'charm'}}selected{{/compare}}" data-type="charm" tabindex="0" role="button">Charms</li>
+            <li class="{{#compare currentType 'bundle'}}selected{{/compare}}" data-type="bundle" tabindex="0" role="button">Bundles</li>
+          </ul>
+        </nav>
+        <div class="six-col last-col">
+          <div class="list-block__filters--selects">
+            <form>
+              <div class="list-block__sort">
+                Sort by:
+                <select>
+                  <option value="-downloads" {{#compare currentSort '-downloads'}}selected="selected"{{/compare}}>Most popular</option>
+                  <option value="downloads" {{#compare currentSort 'downloads'}}selected="selected"{{/compare}}>Least popular</option>
+                  <option value="name" {{#compare currentSort 'name'}}selected="selected"{{/compare}}>Name (a-z)</option>
+                  <option value="-name" {{#compare currentSort '-name'}}selected="selected"{{/compare}}>Name (z-a)</option>
+                  <option value="owner" {{#compare currentSort 'owner'}}selected="selected"{{/compare}}>Author (a-z)</option>
+                  <option value="-owner" {{#compare currentSort '-owner'}}selected="selected"{{/compare}}>Author (z-a)</option>
+                </select>
+              </div>
+              <div class="list-block__series">
+                Series:
+                <select>
+                  {{! The series list has been limited to sane options, full list is here:
+                  https://github.com/juju/charmstore/blob/v5-unstable/internal/router/router.go#L33 }}
+                  {{#each allSeries}}
+                    <option value="{{this}}" {{#compare currentSeries this}}selected="selected"{{/compare}}>{{fullSeries this}}</option>
+                  {{/each}}
+                </select>
+              </div>
+            </form>
+          </div>
+        </div>
+      </div>
+      {{#each currentTopics}}
+        <div class="twelve-col solutions__feature-link">
+          <div class="ten-col no-margin-bottom">
+            {{#if topic.searchIcon}}
+              <img src="{{STATIC_URL}}{{topic.searchIcon}}" alt="{{topic.name}} icon" />
+            {{/if}}
+            <h3 class="solutions__feature-heading">
+              <a href="{{topic.url}}">Juju solutions for {{topic.displayName}}&nbsp;&rsaquo;</a>
+            </h3>
+            <p class="solutions__feature-text">
+              {{#compare id 'openstack'}}
+                Juju makes it easy to deploy OpenStack at scale.  Quickly and reliably build an enterprise-scale cloud run on Ubuntu &mdash; the most popular operating system for OpenStack.
+              {{/compare}}
```

Comment:
So I certainly see this on jujucharms.com if I search for openstack--are we really porting all of that to the GUI? I thought it was just the results part.